<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>565</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="player1Label">
      <property name="minimumSize">
       <size>
        <width>241</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>241</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Player 1</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QPushButton" name="chooseTime1">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>120s</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <spacer name="verticalSpacer_8">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="17" column="1" colspan="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QProgressBar" name="progressBarP1">
      <property name="minimumSize">
       <size>
        <width>241</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>241</width>
        <height>28</height>
       </size>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="15" column="4">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="4" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QPushButton" name="chooseTime2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>5min</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
     <widget class="QLabel" name="gameInfoLabel">
      <property name="minimumSize">
       <size>
        <width>134</width>
        <height>109</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select playtime and start the game</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="5">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="12" column="5">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QProgressBar" name="progressBarP2">
      <property name="minimumSize">
       <size>
        <width>241</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>241</width>
        <height>28</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="16" column="4" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QPushButton" name="stopGame">
      <property name="minimumSize">
       <size>
        <width>241</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>STOP GAME</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="12" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="player2Label">
      <property name="minimumSize">
       <size>
        <width>241</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>241</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Player 2</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="2">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="16" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QPushButton" name="startGame">
      <property name="minimumSize">
       <size>
        <width>241</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>START GAME</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="4" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QPushButton" name="switchPlayer2">
      <property name="minimumSize">
       <size>
        <width>241</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>241</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Switch Player 2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QPushButton" name="switchPlayer1">
      <property name="minimumSize">
       <size>
        <width>241</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>241</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Switch Player 1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
